packages/esbuild-plugin-import-path-remapper/package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "@rnx-kit/esbuild-plugin-import-path-remapper",
+  "version": "1.0.0",
+  "description": "Esbuild plugin for remapping 'lib/' imports to 'src/'",
+  "homepage": "https://github.com/microsoft/rnx-kit/tree/main/packages/esbuild-plugin-import-path-remapper#readme",
+  "license": "MIT",
+  "files": [
+    "src/*"
+  ],
+  "main": "src/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/rnx-kit",
+    "directory": "packages/esbuild-plugin-import-path-remapper"
+  },
+  "scripts": {
+    "build": "rnx-kit-scripts build",
+    "format": "rnx-kit-scripts format",
+    "lint": "rnx-kit-scripts lint",
+    "test": "rnx-kit-scripts test"
+  },
+  "peerDependencies": {
+    "esbuild": "^0.12.19"
+  },
+  "dependencies": {
+    "esbuild": "^0.12.19"
+  },
+  "devDependencies": {
+    "prettier": "^2.0.0"
+  },
```

I don't think you need `esbuild` in `dependencies` or `peerDependencies` since you only need typings from it:
```suggestion
  "devDependencies": {
    "esbuild": "^0.12.19"
  },
```

---

Depcheck complains though.